apps/db/lib/player/stats.ex

```diff
@@ -0,0 +1,19 @@
+defmodule DB.Player.Stats do
+  @moduledoc """
+  This module encapsulates the functionality for player stats.
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  @primary_key :false
+  embedded_schema do
+    field :steps, :integer
+  end
+
+  @fields [:steps]
+
+  def changeset(struct, params) do
```

Comment:
good to get tests going here, even though it's simple now it probably won't stay that way